Library/Formula/zeal.rb

```diff
@@ -0,0 +1,43 @@
+class Zeal < Formula
+  desc "Zeal is a simple offline documentation browser inspired by Dash."
+  homepage "http://zealdocs.org/"
+  url "https://github.com/zealdocs/zeal/archive/v0.1.1.tar.gz"
+  sha256 "f4e959f9bc66a6e350ee8a33d34695379633432d103db9776c0c7a76cbc5a9d6"
+  head "https://github.com/zealdocs/zeal.git"
+
+  depends_on "qt5"
+  depends_on "libarchive"
+
+  patch :DATA
+
+  def install
+    qt5 = Formula["qt5"]
+    ENV.append "CFLAGS", "-I#{qt5.installed_prefix}/include -I#{HOMEBREW_PREFIX}/opt/libarchive/include"
+    ENV.append "CXXFLAGS", "-I#{qt5.installed_prefix}/include -I#{HOMEBREW_PREFIX}/opt/libarchive/include"
+    ENV.append "LDFLAGS", "-L#{qt5.installed_prefix}/lib -L#{HOMEBREW_PREFIX}/opt/libarchive/lib -larchive"
+
+    system "#{qt5.installed_prefix}/bin/qmake"
+    system "make"
+    system "#{qt5.installed_prefix}/bin/macdeployqt", "bin/Zeal.app"
+    prefix.install "bin/Zeal.app"
+    (bin/"zeal").write("#! /bin/sh\n#{prefix}/Zeal.app/Contents/MacOS/Zeal \"$@\"\n")
+  end
+
+  test do
+    system "zeal", "-h"
```

Comment:
Can a test be added to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!